docs/docs/guides/authentication/jwt.md

```diff
@@ -1,5 +1,116 @@
 # `JWT`
 
+## Example Configuration
+
+A minimal example of the configuration for this filter is:
+
+```yaml title="api.yaml"
+apiVersion: gateway.kusk.io/v1alpha1
+kind: API
+metadata:
+  name: auth-jwt-oauth0
+  namespace: default
+spec:
+  fleet:
+    name: default
+    namespace: default
+  spec: |
+    openapi: 3.1.0
+    info:
+      title: "auth-jwt-oauth0"
+      description: "auth-jwt-oauth0"
+      version: "0.1.0"
+    schemes:
+    - http
+    - https
+    x-kusk:
+      auth:
+        jwt:
+          jwtProviders:
```

Comment:
make it just providers it is already under jwt so it should be self-explanatory

Reply:
Renamed to `providers`.

Resolved.